src/tiled.lisp

```diff
@@ -358,6 +392,86 @@
         (gid (tobject-gid tobject))
         (image (tobject-image tobject)))
     (cond
+      (template
+       (let* ((template-object (load-template template))
```

Comment:
I feel like it'd be nice if we could maybe roll this `cond` out to a shared fn to avoid duplicating each object kind for both the template case and the others.
But I'm also not against the more straightforward 'duplicated' version here either.
So, something to consider if you're feeling ambitious but not a dealbreaker at all (and something easily added to in the future)

Reply:
Yep, deduplicating code would be nice, but right now I can't see how it can be done here in an easy way. Let's leave it for later 😅